test/test-framework/pkg/apis/cache/v1alpha1/dummy_types.go

```diff
@@ -0,0 +1,131 @@
+	// +operator-sdk:gen-csv:customresourcedefinitions.statusDescriptors=true
+	Nodes []string `json:"nodes"`
+	// Not included in status but children should be
+	Boss Hog `json:"hog"`
+}
+
+// +k8s:deepcopy-gen=false
+// +k8s:openapi-gen=false
+type Hog struct {
+	// Should be in status but not spec, since Hog isn't in DummySpec
+	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=true
+	// +operator-sdk:gen-csv:customresourcedefinitions.statusDescriptors=true
+	// +operator-sdk:gen-csv:customresourcedefinitions.statusDescriptors.displayName="boss-hog-engine"
+	Engine Engine `json:"engine"`
+	// Not in spec or status, no boolean annotation
+	// +operator-sdk:gen-csv:customresourcedefinitions.statusDescriptors.displayName="doesnt-matter"
+	Brand string `json:"brand"`
+	// Not in spec or status
+	Helmet string `json:"helmet"`
+}
+
+// +k8s:deepcopy-gen=false
+// +k8s:openapi-gen=false
+type Engine struct {
+	// Should not be included, no annotations.
+	Pistons []string `json:"pistons"`
+}
+
+// +k8s:deepcopy-gen=false
+// +k8s:openapi-gen=false
+type OtherDummyStatus struct {
+	// Should be in status but not spec, since this isn't a spec type
+	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=true
+	// +operator-sdk:gen-csv:customresourcedefinitions.statusDescriptors=true
+	Nothing string `json:"nothing"`
+}
+
+// Dummy is the Schema for the dummy API
+// +k8s:deepcopy-gen=false
+// +k8s:openapi-gen=false
+// +kubebuilder:subresource:status
+// +kubebuilder:resource:path=dummys,scope=Namespaced
+// +operator-sdk:gen-csv:customresourcedefinitions.displayName="Dummy App"
+// +operator-sdk:gen-csv:customresourcedefinitions.resources="Deployment,v1,\"dummy-deployment\""
+// +operator-sdk:gen-csv:customresourcedefinitions.resources="ReplicaSet,v1beta2,\"dummy-replicaset\""
+// +operator-sdk:gen-csv:customresourcedefinitions.resources="Pod,v1,\"dummy-pod\""
+type Dummy struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec   DummySpec   `json:"spec,omitempty"`
+	Status DummyStatus `json:"status,omitempty"`
+}
+
+// OtherDummy is the Schema for the other dummy API
+// +k8s:deepcopy-gen=false
+// +k8s:openapi-gen=false
+// +operator-sdk:gen-csv:customresourcedefinitions.displayName="Other Dummy App"
+// +operator-sdk:gen-csv:customresourcedefinitions.resources="Service,v1,\"other-dummy-service\""
+// +operator-sdk:gen-csv:customresourcedefinitions.resources="Pod,v1,\"other-dummy-pod\""
+type OtherDummy struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec   Hog              `json:"spec,omitempty"`
+	Status OtherDummyStatus `json:"status,omitempty"`
+}
+
+// DummyList contains a list of Dummy
+// +k8s:deepcopy-gen=false
+// +k8s:openapi-gen=false
+type DummyList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata,omitempty"`
+	Items           []Dummy `json:"items"`
```

Comment:
```suggestion
	// +listType=set                
	Items           []Dummy `json:"items"`
```

Reply:
@camilamacedo86  I could be wrong, but I'm pretty sure `+listType=set` is not necessary since `// +k8s:openapi-gen=false`

Reply:
I think that if the `+k8s:openapi-gen=false` it is not required as well. 
Anyway, now we have the CI validation for it as well. So, if after the rebase with master it pass in the CI all is fine 👍

Reply:
`+k8s:openapi-gen=false` turns off the openapi generator so any openapi-related tags are not needed. This will pass CI validation.

Reply:
solved